tomcat7/hooks/init

```diff
@@ -25,16 +26,21 @@ debug "pkg.svc_static_path   = {{pkg.svc_static_path}}"
 debug "pkg.svc_var_path      = {{pkg.svc_var_path}}"
 
 # Move directories that ship in the package into place
-cp -a {{pkg.path}}/tc {{pkg.svc_var_path}}/
+# DON'T MOVE the conf dir
+for dir in $(cd {{pkg.path}}/tc/; hab pkg exec core/coreutils ls -1 -I conf)
+do
+  echo "Copying $dir"
+  cp -R {{pkg.path}}/tc/* {{pkg.svc_var_path}}/
+done
+
+# Create the conf directory
+mkdir -p {{pkg.svc_var_path}}/conf
 
 # Symlink config files into $TOMCAT_HOME/conf
 for file in $(cd {{pkg.svc_config_path}}; ls -1)
```

Comment:
does this for loop work for you? I've been having weird stuff happen where the commands within the for loop become concatenated and then the command fails. It's been a while since I've tested, but in my current tomcat builds, I replaced the for loop with explicit lines.

Reply:
If you're looking at the original definition, it's actually not the for loop's fault, the filename in the /conf/ directory was concatenated!! I haven't had any issues since I split out all the files to their correct homes.